roottest/root/graf/testImageFormats.cxx

```diff
@@ -0,0 +1,9 @@
+#include <TGClient.h>
+
+#include <gtest/gtest.h>
+
+TEST(ImageFormats, GIF) {
+  // Import the GoBack icon from ROOT's icon directory
+  auto icon = gClient->GetPicture("GoBack.gif");
+  ASSERT_NE(icon, nullptr);
```

Comment:
Thanks for this!
As seen in https://github.com/root-project/root/issues/19412#issuecomment-3097703307
the return value is not a nullptr even if there is an error when loading the file, so this test will always succeed.
Maybe adding
ASSERT_NE(icon->GetMask(), nullptr);
or capturing diagnostic helps?